Library/Homebrew/cmd/update.rb

```diff
@@ -22,9 +22,11 @@ def update
     migrate_taps
 
     report = Report.new
+    repo_changes = []
     master_updater = Updater.new(HOMEBREW_REPOSITORY)
     master_updater.pull!
     report.update(master_updater.report)
+    repo_changes << ["homebrew/homebrew", *master_updater.revision_range]
```

Comment:
I would have expected `a, b, c = [1, [2, 3]]` to become `a = 1`, `b = [2, 3]`, and `c = nil` and apparently that's also the case. Anyway, thanks for the hint. I'll try to stay away from multiple assignment and find a cleaner solution once I update the PR.